tests/Behat/Element/Admin/Product/ShowPage/PricingElement.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Element\Admin\Product\ShowPage;
+
+use Behat\Mink\Element\NodeElement;
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+use Webmozart\Assert\Assert;
+
+final class PricingElement extends Element implements PricingElementInterface
+{
+    public function getVariantPricingRowForChannel(string $variantName, string $channelName): NodeElement
+    {
+        $pricingRow = $this->getDocument()->find(
+            'css',
+            sprintf('tr:contains("%s") + tr:contains("%s")', $variantName, $channelName),
+        );
```

Comment:
Can be extracted to a defined element, in which case you wouldn't need the `Assert` below.